Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuthManager.m

```diff
@@ -52,19 +52,27 @@ - (instancetype)initWithAppKey:(NSString *)appKey {
 }
 
 - (instancetype)initWithAppKey:(NSString *)appKey host:(nullable NSString *)host {
+  return [self initWithAppKey:appKey host:host redirectURL:nil];
+}
+
+- (instancetype)initWithAppKey:(NSString *)appKey host:(nullable NSString *)host redirectURL:(nullable NSString *)redirectURL {
```

Comment:
Let's keep nullability keywords only in header file

Reply:
Any specific reason? It's called with nil redirectURL above.

Reply:
It's redundant if it's in both header and impl file

Reply:
k that makes sense, removed.